packages/react-components/src/templates/ProfileHeader.tsx

```diff
@@ -84,9 +92,24 @@ const ProfileHeader: React.FC<ProfileProps> = ({
   aboutHref,
   researchHref,
   outputsHref,
+  discoverHref,
+  role,
 }) => {
+  const teamsProps: ProfileProps['teams'] =
```

Comment:
Based on the conversation yesterday, I thought we would solve this by passing the `role` into `ProfilePersonalText` as well and letting it decide what to show for user with role `Staff`